bigframes/core/blocks.py

```diff
@@ -2646,7 +2646,7 @@ def _get_rows_as_json_values(self) -> Block:
         # The only ways this code is used is through df.apply(axis=1) cope path
         # TODO: Stop using internal API
```

Comment:
I don't understand what this TODO is about. I think it's probably out of date. Let's remove it.

```suggestion
```